app/controllers/ChoiceController.scala

```diff
@@ -63,10 +63,11 @@ class ChoiceController @Inject()(
             .update(
               authenticatedSessionId,
               ExportsDeclaration(
+                DeclarationStatus.DRAFT,
                 authenticatedSessionId,
                 UUID.randomUUID().toString,
-                createdDateTime = LocalDateTime.now,
-                updatedDateTime = LocalDateTime.now,
+                createdDateTime = Instant.now,
+                updatedDateTime = Instant.now,
```

Comment:
Just to highlight I've changed the way we store DateTimes. This wont affect existing data as we have a Mongo formatter for them.